resin/assets/test.frag

```diff
@@ -16,8 +13,18 @@ uniform float u_nearPlane;
 uniform float u_farPlane;
 uniform float u_camSize;
 
+#include "blinn_phong.glsl"
+#include "sdf.glsl"
+#external_definition SDF_CODE
+
 // rendering
 const vec3 u_Ambient = vec3(0.25,0.25,0.25);
+const int kMaxNodeCount = 10;
```

Comment:
Could it be injected with `#external_definition`? If so the nodes registry would be the only place when we define the max node count

Reply:
Good catch, completely forgot about his variable x